packages/js/data/src/navigation/index.ts

```diff
@@ -20,15 +20,23 @@ import { PromiseifySelectors } from '../types/promiseify-selectors';
 
 export { type State };
 
+export const INTERNAL_CALL = Symbol( 'INTERNAL_CALL' );
```

Comment:
I wouldn’t keep the variable here because `export * from "./index"` could unintentionally expose `INTERNAL_CALL`. I’d move it into `dispatchers` or a dedicated file to avoid making it public.